tests/testerina-integration-test/src/test/resources/testng.xml

```diff
@@ -37,7 +37,7 @@ under the License.
             <class name="org.ballerinalang.testerina.test.negative.SkipTestsTestCase" />
             <class name="org.ballerinalang.testerina.test.negative.InvalidDataProviderTestCase" />
             <class name="org.ballerinalang.testerina.test.DisableTestsTestCase" />
-            <class name="org.ballerinalang.testerina.test.MockTest" />
+<!--            <class name="org.ballerinalang.testerina.test.MockTest" />-->
```

Comment:
Why we comment this?

Reply:
Uncommented them and commented the email relevant tests. Will open issues as discussed with Joe.

Reply:
The issue created is: https://github.com/ballerina-platform/ballerina-lang/issues/25514